float32 servoFoil
float32 servoGouvernail
float32 servoAileronLeft
float32 servoAileronRight
float32 thruster
